notebooks/00_quick_start/run_notebook_on_aml.ipynb

```diff
@@ -14,121 +14,93 @@
    "metadata": {},
```

Comment:
Title should not contain aml - please use azureml

Reply:
following the same idea, the name of the file would be `run_notebook_on_azureml.ipynb`

Reply:
Changed now